pages/alpha/sponsorship/index.tsx

```diff
@@ -69,27 +75,21 @@ const Sponsorship: NextPage = () => {
               </b>
             </h1>
             <h3 className="text-lg font-bold">
-              Reach thousands of developers every mounth!
+              Reach thousands of developers every month!
             </h3>
           </header>
           <section className="flex items-center relative bottom-12 justify-center overflow-hidden gap-8 sm:gap-20 sm:bottom-20 md:bottom-24  md:gap-36 lg:gap-44">
-            {images.map((image, id) => (
+            {images.map((image) => (
               <div
-                key={id}
+                key={image.alt}
                 className={`w-32`}
                 style={{
-                  transform: `${
-                    scroll < 200
-                      ? `rotate(${(scroll / 200) * image.rotate}deg)`
-                      : `rotate(${image.rotate}deg)`
-                  }`,
+                  transform: `rotate(calc(min(var(--scroll), 200) / 200 * ${image.rotate}deg))`
                 }}
               >
                 <Image
                   src={image.src}
                   alt={image.alt}
-                  width={250}
```

Comment:
@NiallJoeMaher It does. According to [the doc](https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images), when using locally imported images...
> Next.js will [automatically determine](https://nextjs.org/docs/app/building-your-application/optimizing/images#image-sizing) the width and height of your image based on the imported file. These values are used to prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) while your image is loading.

However, image dimensions slightly change, because now their true dimension `268x268` is being used, instead of `250x250` as specified in the code.

**Before**
<img width="1373" alt="Screenshot 2023-06-28 at 10 23 51 p m" src="https://github.com/codu-code/codu/assets/28614996/2c330e3f-5e84-49e4-a885-462b6f46d7b4">

**After**
<img width="1373" alt="Screenshot 2023-06-28 at 10 22 33 p m" src="https://github.com/codu-code/codu/assets/28614996/30c3d2de-1989-412d-b122-8c13b0d30fa3">

Reply:
If we prefer the fixed `250x250`, I can add that back

Reply:
I want to add that on client side, those images are converted to very lightweight `webp` (9x smaller than original images). This was already the case before, my change turn out to not bring additional optimization. 
<img width="1375" alt="Screenshot 2023-06-28 at 10 18 17 p m" src="https://github.com/codu-code/codu/assets/28614996/6263a493-cdb7-48a1-821c-8c54c5916af3">

So the chunky and slow feeling mentioned in #285 could solely be excessive re-rendering. Using CSS transformation should address this.